### PR TITLE
Feature/create region value

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**

### CODE REVIEW

**Review by maxcapodi78 — DISMISSED** [review]
If possible I would add a new argument to the the method: is_percentage. If true I consider the values as percentage, otherwise as absolute dimension. in this way I can still pass float to absolute dimension (in modeler units)

File: pyaedt/modeler/Modeler.py [hunk 1]
```diff
@@ -3609,20 +3609,30 @@ def create_air_region(self, x_pos=0, y_pos=0, z_pos=0, x_neg=0, y_neg=0, z_neg=0
...
+            If a float, use padding in percent in the +Y direction.
+            If a str, use padding in absolute value in the +Y direction.
+            The default is ``0``.
+        z_pos : float or str, optional
+            If a float, use padding in percent in the +Z direction.
+            If a str, use padding in absolute value in the +Z direction.
+            The default is ``0``.
+        x_neg : float or str, optional
+            If a float, use padding in percent in the +Z direction (-R for 2D RZ).
+            If a str, use padding in absolute value in the +Z direction (-R for 2D RZ).
```
Comment (MaxJPRey) [inline pyaedt/modeler/Modeler.py:3626]:
```suggestion
            If a float, use padding in percent in the -X direction (-R for 2D RZ).
            If a str, use padding in absolute value in the -X direction (-R for 2D RZ).
```

File: pyaedt/modeler/Primitives.py [hunk 2]
```diff
@@ -1304,9 +1304,9 @@ def create_region(self, pad_percent=300):
 
         Parameters
         ----------
-        pad_percent : float or list of floats, optional
+        pad_percent : float or list of floats or list of str, optional
```
Comment (PipKat) [inline pyaedt/modeler/Primitives.py:1307]: I thought when different data formats were acceptable, you listed them with commas between the names and not "or"--so: "float, list of floats, list of str".

**Comment by Samuelopez-ansys** [comment]
Hi @PipKat  and @MaxJPRey ,

I had to modify most of the original PR, please could you review the style again?

Thank you so much

**Comment by MaxJPRey** [comment]
> Hi @PipKat and @MaxJPRey ,
> 
> I had to modify most of the original PR, please could you review the style again?
> 
> Thank you so much

Sure, will do.

**Review by MaxJPRey — APPROVED** [review]

**Pull request merged**